app/models/talk.rb

```diff
@@ -97,4 +99,15 @@ def self.propose_upcoming_months
   def happened?
     happened_at && happened_at < Date.today
   end
+
+  private
+
+  def send_slack_notification
+    conn = Faraday.new(headers: {'Content-Type' => 'application/json'}) do |conn|
```

Comment:
Send the notification synchronously because we have no sidekiq dyno running (this may be the reason mail are not sent too, I'll ask yanski if we can add a sidekiq dyno)

Reduced the timeout to 5 sec so that the page can't be blocked and a rescue all to avoid errors (we will only miss the slack notification on this case)